app/containers/GameBoard.jsx

```diff
@@ -39,23 +39,48 @@ class GameBoard extends Component {
   createGame() {
     const deck = createDeck();
     const shuffled = shuffleDeck(deck);
+    // in this case, I wouldn't pass this.state.numOfPlayers
+    // to dealDeck as a parameter since your dealDeck
+    // method has access to that. it will be cleaner that way.
+    // You want to parameterize things moreso when you want 
+    // to extract that function or method from this class
     this.dealDeck(shuffled, this.state.numOfPlayers)
   }
 
+  // name your variables and parameters as specifically as you can
+  // array is too vague
   dealDeck(array, numOfPlayers) {
     const deck1 = [];
     const deck2 = [];
     const deck3 = [];
-    const deck4 = [];
+    const deck4 = []; 
+    /*
+      suggestion part one
+      const decks = {};
+    */
     let i = 1;
     while (array.length > 0) {
       const card = array.pop();
+      /*
+        suggestion part two
+        const currentDeck = decks[`deck{i}`]
+        if (currentDeck === undefined) {
+          currentDeck = [card];
+        } else {
+          currentDeck.push(card);
+        }
+      */
+      // This is a little too "brute force"
+      // See suggestions part 1 and two above
```

Comment:
I felt it was very brute force too. I like how you used an object and simplified the logic.